.github/workflows/google_export.yaml

```
@@ -0,0 +1,30 @@
+on:
+  workflow_dispatch:  # 手動トリガー
+  schedule:
+    - cron: '0 6 * * *' # 毎日06:00 UTCに実行（日本時間15:00）
+
+jobs:
+  export:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)